scripts/script.js

```diff
@@ -0,0 +1,57 @@
+//index.html & dogs.html click handlers for DOG PHOTOS
+
+function murphyInfo() {
+		alert("Murphy, Beagle, $123.45");
+}
+
+function poppyInfo() {
+		alert("Poppy, Shih Tzu, $123.45");
+}
+
+function jackInfo() {
+		alert("Jack, Jack Russel Terrier, $123.45");
+}
+
+function duffyInfo() {
+		alert("Duffy, Pitbull, $123.45");
+}
+
+function lucasInfo() {
+		alert("Lucas, Beagle, $123.45");
+}
+
+function jakeInfo() {
+		alert("Jake, Labrador, $123.45");
+}
+
+function angusInfo() {
+		alert("Angus, Boxer, $123.45");
+}
+
+function violetInfo() {
+		alert("Violet, Chocolate Labrador, $123.45");
+}
+
+function piperInfo() {
+		alert("Piper, Chocolate Labrador, $123.45");
+}
+
+function maximusInfo() {
+		alert("Maximus, Chocolate Labrador, $123.45");
+}
+
+function lunaInfo() {
```

Comment:
Good job, but you'll want to do this in a different way. Write one function that takes info from the HTML and passes it into the function. That way you only need one function.

example...

```
<button onclick="thingDoer('james', 42)">click</button>
```
then, your function
```
function thingDoer(name, age) {
  alert(name + " is " + age)
}
```